src/styles/utils/variables.scss

```
@@ -0,0 +1,12 @@
+$shadow-header: rgba(0, 0, 0, 0.05);
+$bg-is-active: #00ACDC;
+$text-color: #060B35;
+$subtitle-color: #616070;
+$bg-color: #FFF;
+$bg-color-button: #00ACDC;
```

No need to duplicate variables with the same values
```suggestion
$bg-is-active: #00ACDC;
$text-color: #060B35;
$subtitle-color: #616070;
$bg-color: #FFF;
```